staging/src/k8s.io/cri-api/pkg/apis/runtime/v1alpha2/api.proto

```diff
@@ -279,13 +279,37 @@ message LinuxSandboxSecurityContext {
     // This allows a sandbox to take additional security precautions if no
     // privileged containers are expected to be run.
     bool privileged = 6;
+    // Seccomp profile for the sandbox.
+    SecurityProfile seccomp_profile = 9;
+    // AppArmor profile for the sandbox.
+    SecurityProfile apparmor_profile = 10;
```

Comment:
@mikebrow @mrunalp PTAL, this is a slightly different Approach where we can re-use the same type `SecurityProfile` for seccomp and AppArmor.